WordPress/src/main/java/org/wordpress/android/designsystem/DesignSystemDataSource.kt

```diff
@@ -3,8 +3,16 @@ package org.wordpress.android.designsystem
 import org.wordpress.android.R
 
 object DesignSystemDataSource {
```

Comment:
How about modeling this using data classes, and Sealed class/interface

Reply:
So I think using an `enum` here would make sense here since these are fixed values. Could you elaborate more on why a sealed class might be better? From what I understand, sealed classes are for a set of classes whereas an enum would be better for a set of fixed values

Reply:
Probably it is a good idea to introduce a stateholder like ViewModel, and model this data provider in the next PR